src/iterate.js

```diff
@@ -151,6 +163,10 @@ class Iterate extends events.EventEmitter {
       // Wait until we've done all the checks to emit success
       this.emit('iteration-success', value);
       debug(`ran handler in ${diff} seconds`);
+      if (this.monitor) {
+        this.monitor.measure('iteration-time', diff * 1000);
+        this.monitor.count('successful-iteration', 1);
+      }
```

Comment:
Basically, we want to record in ms everywhere to keep things consistent... I guess it could be crazy...

/me sad things doesn't support units...